### PR TITLE
Some virus2 logging

### PR DESCRIPTION
**Pull request by despotate**
- Adds a bit more virus2 `debug_log`s, along with a bunch of commented out logs for when we want more resolution
- Changes `badness` to `max_badness` in a `disease` proc for clarity
- A bunch of spaces got removed and I had nothing to do with it, must've been atom

Tested, logs were fairly sparse with viruses after infecting myself and some monkeys

### CODE REVIEW

File: code/modules/virus2/disease2.dm [hunk 1]
```diff
@@ -119,11 +124,13 @@ var/global/list/disease2_list = list()
 	//Moving to the next stage
 	if(clicks > stage*100 && prob(10))
 		if(stage == max_stage)
+			log_debug("Virus [uniqueID] in [key_name(mob)] has advanced past its last stage, giving them antibodies [antigens2string(antigen)].")
 			src.cure(mob)
 			mob.antibodies |= src.antigen
 			log += "<br />[timestamp()] STAGEMAX ([stage])"
 		else
 			stage++
+			// log_debug("Virus [uniqueID] in [key_name(mob)] has advanced to stage [stage].")
```
Comment (Probe1) [inline code/modules/virus2/disease2.dm]: This is going to spam logs like crazy.

Comment (Probe1) [inline code/modules/virus2/disease2.dm]: Why add a commented out line

File: code/modules/virus2/disease2.dm [hunk 2]
```diff
@@ -152,6 +160,7 @@ var/global/list/disease2_list = list()
 	var/datum/disease2/effect/e = pick(effects)
 	e.minormutate()
 	infectionchance = min(50,infectionchance + rand(0,10))
+	// log_debug("Virus [uniqueID] has minor mutated [e.name].")
```
Comment (Probe1) [inline code/modules/virus2/disease2.dm]: ditto here

**Comment by despotate** [comment]
I added them as logs but then commented them out when you said the logs were too verbose
we can just keep 'em there for when someone wants to make a change to virus2 and might want the extra logging

**Comment by Probe1** [comment]
Lets add them then and not predictive commented stuff please.

**Comment by despotate** [comment]
But you previously said that they were way too much detail when I showed you the logs that were generated by infecting five monkeys. I have three options:

- Remove the commented lines entirely
- Keep them commented out
- Uncomment them (you've said they're too much detail)

What do you want me to do?

**Comment by Probe1** [comment]
Delete them.

**Comment by Probe1** [comment]
I have a feeling that this will be a little too much.  But on the other hand I'm gonna merge it and see how it plays out.

**Pull request merged**